### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by junketjim**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/junketjim/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
```
Comment (junketjim) [inline ReadMe.md]: What's this? !fix

Comment (junketjim) [inline ReadMe.md]: !resolve

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
```
Comment (junketjim) [inline ReadMe.md]: This is a line note

File: ReadMe.md (hunk 1 above)
Comment (junketjim) [inline ReadMe.md]: !fixed

**Pull request merged**